src/docfx/lib/GlobUtility.cs

```diff
@@ -49,12 +49,7 @@ bool IsMatch(string path)
             }
         }
 
-        private static bool IsFileStartingWithDot(string path)
-        {
-            return path.StartsWith('.') || path.Contains("/.") || path.Contains("\\.");
-        }
-
-        private static Glob CreateGlob(string pattern)
+        public static Glob CreateGlob(string pattern)
```

Comment:
consider use `CreateGlobMatcher` method in line 12, I want to hide the usage of `Glob` inside this utility.